test/jdk/sun/security/ssl/SignatureScheme/SigAlgosExtTestWithTLS13.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2021 THL A29 Limited, a Tencent company. All rights reserved.
+ * Copyright (C) 2021, 2023 THL A29 Limited, a Tencent company. All rights reserved.
```

Comment:
Missing comma.

Reply:
May be you can revert this file, change is unintentional.

Reply:
reverted